src/app/common/SummaryView.vue

```diff
@@ -10,13 +10,21 @@
     data-testid="summary"
     @close="emit('close')"
   >
+    <template #title>
+      <XTeleportSlot :name="id" />
```

Comment:
**Question**: Why do we need to keep the title of summary views slotted into AppView in the first place? Shouldn’t we just move the to be slotted into KSlideout directly?

Reply:
Well one reason is all our `*View.vue`s look the same, there are no "special" ones just because it goes in a side panel. I suppose this is the biggest reason but there are also a couple of others.